puppet/lib/puppet/util/puppetdb.rb

```diff
@@ -64,6 +66,22 @@ def submit_command(certname, payload, command_name, version)
     end
   end
 
+  # Query PuppetDB.
+  #
+  # @param query [String] The PQL or AST query for PuppetDB
```

Comment:
`[String, Array]` ... right? I mean, it takes an array for AST, and a string for PQL, so it could be either?